config/config.go

```diff
@@ -19,7 +19,7 @@ const (
 	// DefaultPathName is the default config dir name
 	DefaultPathName = ".reference-provider"
 	// DefaultPathRoot is the path to the default config dir location.
-	DefaultPathRoot = "~/" + DefaultPathName
+	DefaultPathRoot = "~/" + DefaultPathName // TODO this won't work on windows. use user home dir instead
```

Comment:
//cc @gammazero, I don't know if this has been fixed, but we had this same issue in storetheindex. I don't expect this or storetheindex to run in Windows, but just in case (and to make the CI gods happy :) )